Geometry/TrackerNumberingBuilder/src/GeometricDet.cc

```diff
@@ -124,54 +133,54 @@ GeometricDet::GeometricDet(const PGeometricDet::Item& onePGD, GeometricEnumType
            onePGD._a31,
            onePGD._a32,
            onePGD._a33),
-      _shape(cms::dd::name_from_value(cms::LegacySolidShapeMap, static_cast<LegacySolidShape>(onePGD._shape))),
-      _ddd(),
-      _ddname(onePGD._name),
-      _type(type),
-      _params(),
-      _geographicalID(onePGD._geographicalID),
-      _radLength(onePGD._radLength),
-      _xi(onePGD._xi),
-      _pixROCRows(onePGD._pixROCRows),
-      _pixROCCols(onePGD._pixROCCols),
-      _pixROCx(onePGD._pixROCx),
-      _pixROCy(onePGD._pixROCy),
-      _stereo(onePGD._stereo),
-      _siliconAPVNum(onePGD._siliconAPVNum) {
+      shape_(cms::dd::name_from_value(cms::LegacySolidShapeMap, static_cast<LegacySolidShape>(onePGD._shape))),
+      ddd_(),
+      ddname_(onePGD._name),
+      type_(type),
+      params_(),
+      geographicalID_(onePGD._geographicalID),
+      radLength_(onePGD._radLength),
+      xi_(onePGD._xi),
+      pixROCRows_(onePGD._pixROCRows),
+      pixROCCols_(onePGD._pixROCCols),
+      pixROCx_(onePGD._pixROCx),
+      pixROCy_(onePGD._pixROCy),
+      stereo_(onePGD._stereo),
+      siliconAPVNum_(onePGD._siliconAPVNum) {
   if (onePGD._shape == 1 || onePGD._shape == 3) {  //The parms vector is neede only in the case of box or trap shape
-    _params.reserve(11);
-    _params.emplace_back(onePGD._params0);
-    _params.emplace_back(onePGD._params1);
-    _params.emplace_back(onePGD._params2);
-    _params.emplace_back(onePGD._params3);
-    _params.emplace_back(onePGD._params4);
-    _params.emplace_back(onePGD._params5);
-    _params.emplace_back(onePGD._params6);
-    _params.emplace_back(onePGD._params7);
-    _params.emplace_back(onePGD._params8);
-    _params.emplace_back(onePGD._params9);
-    _params.emplace_back(onePGD._params10);
+    params_.reserve(11);
+    params_.emplace_back(onePGD._params0);
+    params_.emplace_back(onePGD._params1);
+    params_.emplace_back(onePGD._params2);
+    params_.emplace_back(onePGD._params3);
+    params_.emplace_back(onePGD._params4);
+    params_.emplace_back(onePGD._params5);
+    params_.emplace_back(onePGD._params6);
+    params_.emplace_back(onePGD._params7);
+    params_.emplace_back(onePGD._params8);
+    params_.emplace_back(onePGD._params9);
+    params_.emplace_back(onePGD._params10);
   }
 
-  _ddd.reserve(onePGD._numnt);
-  _ddd.emplace_back(onePGD._nt0);
-  _ddd.emplace_back(onePGD._nt1);
-  _ddd.emplace_back(onePGD._nt2);
-  _ddd.emplace_back(onePGD._nt3);
+  ddd_.reserve(onePGD._numnt);
```

Comment:
@ghugo83 - please, consider:
```c++
 auto nt = fv->navPos();
 ddd_ = nav_type(nt.begin(), nt.end());
```
https://cmssdt.cern.ch/lxr/source/DetectorDescription/DDCMS/interface/DDFilteredView.h#0206

Reply:
Hmm here, this is the constructor from PGeometricDet, not from the filtered view.
What you mention is done in the constructor from filtered view.

Reply:
@ghugo83 - you are right, I take it back :-)